rskj-core/src/main/java/co/rsk/peg/BridgeSerializationUtils.java

```diff
@@ -105,23 +102,27 @@ public static SortedMap<Keccak256, BtcTransaction> deserializeMap(byte[] data, N
         return map;
     }
 
-    public static byte[] serializeUTXOList(List<UTXO> list) throws IOException {
+    public static byte[] serializeUTXOList(List<UTXO> list) {
         int nutxos = list.size();
 
         byte[][] bytes = new byte[nutxos][];
         int n = 0;
 
-        for (UTXO utxo : list) {
-            try (ByteArrayOutputStream ostream = new ByteArrayOutputStream()) {
-                utxo.serializeToStream(ostream);
-                bytes[n++] = RLP.encodeElement(ostream.toByteArray());
+        try {
+            for (UTXO utxo : list) {
+                try (ByteArrayOutputStream ostream = new ByteArrayOutputStream()) {
+                    utxo.serializeToStream(ostream);
+                    bytes[n++] = RLP.encodeElement(ostream.toByteArray());
+                }
             }
+        } catch (IOException ioe) {
+            throw new SerializationException("Unable to serialize UTXO list ", ioe);
```

Comment:
The utxo list as string can be part of the exception message as well. Wdyt?

Reply:
good idea! Thanks